models/helpers.ts

```diff
@@ -225,6 +376,26 @@ function getSubmittableDocStatus(doc: RenderData | Doc) {
 }
 
 export function getInvoiceStatus(doc: RenderData | Doc): InvoiceStatus {
```

Comment:
This is too overloaded now. All of the other states should have a different UI element and should not be in place of the common status pill.